src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,37 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.constants.ApplicationConstants;
+import com.elixr.poc.exception.NoRecordFoundException;
+import com.elixr.poc.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
```

Comment:
Wildcard import is not recommended, Please import only required

Reply:
Updated.